src/CodeGen_LLVM.cpp

```diff
@@ -270,6 +295,7 @@ void CodeGen_LLVM::initialize_llvm() {
             for (const std::string &s : arg_vec) {
                 c_arg_vec.push_back(s.c_str());
             }
+            //            c_arg_vec.push_back("-opaque-pointers");
```

Comment:
Left in accidentally?

Reply:
Should not be committed, but good to have it appear here briefly as documentation on how to test with LLVM in opaque pointer mode during its transition. Will remove or comment.

Reply:
+1 to leave-in-with-comment